test/sql/admin.sql

```diff
@@ -0,0 +1,11 @@
+LOAD 'pg_diffix';
+
+-- Mark, unmark objects, show their labels
+CALL diffix.mark_role('diffix_test', 'direct');
+SELECT * FROM diffix.show_labels();
+CALL diffix.unmark_table('public.test_products');
+CALL diffix.unmark_role('diffix_test');
+SELECT * FROM diffix.show_labels();
+CALL diffix.mark_role('diffix_test', 'anonymized_trusted');
+CALL diffix.mark_public('public.test_products');
+SELECT * FROM diffix.show_labels();
```

Comment:
Good for having this file around. `validation.sql` was getting bloated. It still is I think.